### PR TITLE
Doc/750 add style

### PR DESCRIPTION
**Pull request by Moneyba**
## Description
Add style using [docsify](https://docsify.js.org/#/).

## Changes
Restructure the documentation and add style.

### CODE REVIEW

**Comment by davidverdu** [comment]
LGTM but, why do you delete so many lines from package-lock.json file? Another weird thing is that unit tests are not passing

**Review by AdrianLorenzoDev — APPROVED** [review]
LGTM!!! 🙂

File: docs/chapters/02_getting-started.md [hunk 1]
```diff
@@ -153,13 +155,13 @@ To do so, we have two possible solutions:
...
 ```ini
- ~/.aws/credentials
+  ~/.aws/credentials
 [default]
 aws_access_key_id = <DEFAULT ACCESS KEY ID>
 aws_secret_access_key = <DEFAULT SECRET ACCESS KEY>
 region=<DEFAULT REGION>
 
-[other_profile]  Give this profile the name that works best for you
+[other_profile]   Give this profile the name that works best for you
```
Comment (davidverdu) [inline docs/chapters/02_getting-started.md:164]: why do we have this spaces at the beginning of the sentences? " Give this profile..." in place of "Give this profile...". This repeats in many parts of the files

File: docs/chapters/02_getting-started.md [hunk 2]
```diff
@@ -306,7 +373,7 @@ boosted-blog
 
 Now open the project in your favorite editor, e.g. [Visual Studio Code](https://code.visualstudio.com/).
 
-### 2. First command
+###  2. First command
```
Comment (davidverdu) [inline docs/chapters/02_getting-started.md:376]: Anocher white space I don't understand

File: docs/chapters/05_going-deeper.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+
+> **Note:** So far, there is only one provider fully supported in Booster yet, @boostercloud/framework-provider-aws, and it is probably the one you have already set if you used the generator to create your project. Azure and Kubernetes have experimental support and the team is currently improving them.
+
+- **assets**: This is an array of _relative_ paths from the root of the project pointing to files and folders with static assets. They will be included among the deployed files to the cloud provider.
+  For example, imagine you are using the `dotenv` module so that all the environment variables you have in your `.env` files are loaded into memory in runtime. In order for this to work, you need to include your `.env` files as assets of your project, so that they are included when deploying. Assuming you only have a `.env` file in the root of your project, you should add the following to your configuration:
+  ```typescript
+  config.assets = ['.env']
+  ```
+
+# Providers configuration
```
Comment (davidverdu) [inline docs/chapters/05_going-deeper.md:77]: I believe here it should be `### Providers configuration`, don't you think? The doc section is being displayed wrongly in docsify due to this typo

File: docs/chapters/02_getting-started.md [hunk 4]
```diff
@@ -0,0 +1,784 @@
...
+folder, and a file inside called `credentials` with this syntax:
+
+```ini
+  ~/.aws/credentials
+[default]
+aws_access_key_id = <YOUR ACCESS KEY ID>
+aws_secret_access_key = <YOUR SECRET ACCESS KEY>
+```
+
+> You can see more details in the [Providers configuration](chapters/05_going-deeper.md#providers) section.
```
Comment (davidverdu) [inline docs/chapters/02_getting-started.md:147]: It should point to chapters/05_going-deeper.md#providers-configuration instead

File: docs/chapters/02_getting-started.md [hunk 5]
```diff
@@ -0,0 +1,784 @@
...
+
+```shell
+az login
+```
+
+You can check the different login options in this [reference document](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli).
+
+Once you are logged in from the CLI, you can deploy applications with Booster.
+
+> You can see more details in the [Providers configuration](chapters/05_going-deeper.md#providers) section.
```
Comment (davidverdu) [inline docs/chapters/02_getting-started.md:218]: It should point to chapters/05_going-deeper.md#providers-configuration instead

File: docs/chapters/02_getting-started.md [hunk 6]
```diff
@@ -0,0 +1,784 @@
...
+
+Additionally, it is also required to install **kubectl** and [Helm](https://helm.sh). The required **Helm** version is 3 or greater.
+
+Please note that the desired cluster should be accessible from the kubectl command and you can successfully run:
+
+```shell
+kubectl get pods -A
+```
+
+> You can see more details in the [Providers configuration](chapters/05_going-deeper.md#providers) section.
```
Comment (davidverdu) [inline docs/chapters/02_getting-started.md:239]: It should point to chapters/05_going-deeper.md#providers-configuration instead

File: docs/chapters/03_booster-architecture.md [hunk 7]
```diff
@@ -0,0 +1,750 @@
...
+  public static reduceProductRemoved(event: ProductRemoved, currentCart?: Cart): Cart {
+    const newItems = removeFromCart(event.item, currentCart)
+    return new Cart(event.cartID, newItems)
+  }
+}
+```
+
+You can visualize reduction like this:
+
+![reducer process gif](img/reducer.gif)
```
Comment (davidverdu) [inline docs/chapters/03_booster-architecture.md:569]: This image is not visualizing correctly in Docsify

File: docs/chapters/05_going-deeper.md [hunk 8]
```diff
@@ -0,0 +1,367 @@
+📝[Edit on github](https://github.com/boostercloud/booster/blob/main/docs/chapters/going-deeper.md)
+
+# Going deeper with Booster
+
```
Comment (davidverdu) [inline docs/chapters/05_going-deeper.md:4]: here I would add as originally this text, but linking to our new contributing guide:

`Contributing

If you want to start making contributions to Booster, we strongly recommend that you read our contributing guide.`

**Review by davidverdu — CHANGES_REQUESTED** [review]
LGTM. Good job by turning normal documentation into this amazing doc displaying! Nonetheless, I left some minor comments that should be addressed before merging

**Review by davidverdu — APPROVED** [review]
LGTM!

**Pull request merged**